### ISSUE TITLE
netscan --output=xlsx fails

### ISSUE THREAD

**Issue by JohnEbert** [volatilityfoundation/volatility#266]
I have two exhibits, from different computers and users, of nearly identical Windows volatility-2.5.standalone failure when using netscan --output=xlsx      The command-line output as text to screen or file is fine as long as we do not ask for xlsx format using --output=xlsx. See the two exhibit screenshots at https://mega.nz/#F!YRoGCCYK!CfhcFhjG7QIuArUBoYOSug
I will leave them up until the issue is closed.

**Comment by iMHLv2** [comment]
Ah, so according to your screenshot, an exception was raised (openpyxl.utils.exceptions.IllegalCharacterError). Apparently netscan is generating some text that isn't valid in xlsx files (probably the name field of a process that's exited). Can you run netscan with --output=text --output-file=netscan.txt and attach the result here, so I can see which (if any) characters may be invalid?

**Comment by JohnEbert** [comment]
I uploaded 4 more files for your review. The requested --output=text is named sybhaqre.txt and I suspect the offending text is the gibberish date stamp for the IPv6 entry. https://mega.nz/#F!YRoGCCYK!CfhcFhjG7QIuArUBoYOSug

I also uploaded 2 other exhibits that demonstrate another problem with the --output-xlsx format option, where all three show the hex offset is not being formed as hex notation, but signed scientific notation. One file is from the Windows 7 sybhaqre.mem image, asking for xlsx output for the pslist plugin, and the other is from a Windows 10 memdump asking for an xlsx output for the netscan plugin. They both have borked non-hex offset values.

The remaining file attempted to replicate my other users xlsx request output for pslist, but it hung for me and I had to ctl-c out. You will see a line 24 message in this screen similar to the very first netscan xlsx screen. The command line was: volatility-2.5.standalone.exe --filename=sybhaqre.mem --profile=Win7SP1x64 --kdbg=0x2bf70a0 pslist --output=xlsx --output-file=sybhaqrepslist.xlsx

In summary, it seems the module that produces xlsx has at least two different issues, one with offset calculation and one with abnormal date stamp strings (produced by IPv6 strings?)
